punjab/jabber.py

```diff
@@ -103,9 +157,12 @@ def associateWithStream(self, xs):
             init.required = required
             xs.initializers.append(init)
 
-    def _reset(self):
+    def _reset(self, shallow=False):
         # need this to be in xmlstream
-        stream = domish.elementStream()
```

Comment:
Thanks for noticing this. I've added elementStream() func, almost the same as in domish.py. I can also make ShallowSaxElementStream, but not sure if it is necessary.